keyboards/planck/keymaps/gunp/upload-and-make.sh

```diff
@@ -2,15 +2,15 @@
 
 pushd ~/git-repos/qmk_firmware/
 rm -rf keyboards/planck/keymaps/gunp/
-cp -rf ~/Documents/qmk-planck/ keyboards/planck/keymaps/gunp/
-if [[ $1 == "dfu" ]]; then
+cp -rf ~/git-repos/gunp-geek-repo/qmk-planck/ keyboards/planck/keymaps/gunp/
+if [[ $2 == "dfu" ]]; then
     # sudo dfu-programmer atmega32u4 erase --force
     # sudo dfu-programmer atmega32u4 flash .build/planck_rev4_gunp.hex
     # sudo dfu-programmer atmega32u4 reset
-    sudo make planck/rev4:gunp:dfu
+    sudo make planck/$1:gunp:dfu
     # bash ~/bin/gunp-kb-layout-reset
 else
-    make planck/rev4:gunp:clean
-    make planck/rev4:gunp:all
+    make planck/$1:gunp:clean
```

Comment:
You could actually do this on one line. 

Specifically, you could use: 

```sh
make clean planck/$1:gunp
```

This will run the clean command, and then compile the firmware.